src/Platform/AcquiaCloudPlatform.php

```diff
@@ -182,21 +185,44 @@ public static function getEnvironmentQuestion(Config $config, AcquiaCloudClientF
    * {@inheritdoc}
    */
   public function execute(Command $command, InputInterface $input, OutputInterface $output) : int {
-    $environments = new Environments($this->getAceClient());
     $sites = $this->getPlatformSites();
     if (!$sites) {
       $output->writeln('<warning>No sites available. Exiting...</warning>');
       return 1;
     }
 
+    $group_name = $input->getOption('group');
     $input_uri = $input->getOption('uri');
-    $sites = array_column($sites, 'uri');
-    $args = $this->dispatchPlatformArgumentInjectionEvent($input, $sites, $command);
+
+    if ($group_name && $input_uri) {
+      $helper = $command->getHelper('question');
+      $question = new ConfirmationQuestion('You have provided both the options, group as well as uri. We will ignore the uri option. Do you want to proceed (y/n)?', TRUE);
```

Comment:
This is problematic when a command runs internally through PlatformCommand Executioner in which case user isn't shown this question and execution kind of halts without informing anything to the user.

Reply:
Let's ignore group and only use uri in this case without any choice question.

Reply:
Let's ignore group and only use uri in this case without any choice question.

Reply:
Done.